### PR TITLE
W-10169901 Published Grants-Management 1.19 installers

### PR DESCRIPTION
**Pull request by spelak-salesforce**
[W-10169901](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00000Oaj0nYAB/view)

# Critical Changes

# Changes
Published Grants-Management 1.19 installers.

# Issues Closed

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,143 @@
+# Grants Management Installers
```
Comment (melissabarber) [inline README.md]:
```suggestion
# Grants Management Express Setup
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+
+Welcome to the Grants Management installers!
```
Comment (melissabarber) [inline README.md]:
```suggestion
```

File: README.md [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+
+Grants Management streamlines the grantmaking process and helps grantmakers cultivate and manage relationships with their grantees. Leveraging Salesforce Communities, Grants Management makes grantmaking a breeze through an intuitive, easy-to-use interface and backend automation.
```
Comment (melissabarber) [inline README.md]:
```suggestion
This read-only repository contains CumulusCI automation and unmanaged metadata components used in the Grants Management MetaDeploy installer.
```

File: README.md [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+
+_IMPORTANT_: Salesforce does not recommend installing upgrades directly in production organizations. Instead, install Grants Management in one of your free sandbox environments or a developer organization first.
```
Comment (melissabarber) [inline README.md]:
```suggestion
```

File: README.md [hunk 5]
```diff
@@ -0,0 +1,143 @@
...
+
+Once you've installed Grants Management, check out these resources:
+
+-   [Grants Management Documentation](https://powerofus.force.com/s/article/GM-Documentation)
+-   [Salesforce Partner Community - Grants Management Experts Group](https://partners.salesforce.com/_ui/core/chatter/groups/GroupProfilePage?g=0F94V000000LpYeSAK)
```
Comment (melissabarber) [inline README.md]:
```suggestion
Visit the [Grants Management customer group](https://trailhead.salesforce.com/trailblazer-community/groups/0F94S000000GvrgSAC) or [Salesforce Partner Community - Grants Management Experts Group](https://partners.salesforce.com/_ui/core/chatter/groups/GroupProfilePage?g=0F94V000000LpYeSAK) for any questions about this repository or Grants Management.
```

File: README.md [hunk 6]
```diff
@@ -0,0 +1,143 @@
+# Grants Management Installers
+
+Welcome to the Grants Management installers!
+
+Grants Management streamlines the grantmaking process and helps grantmakers cultivate and manage relationships with their grantees. Leveraging Salesforce Communities, Grants Management makes grantmaking a breeze through an intuitive, easy-to-use interface and backend automation.
+
+_IMPORTANT_: Salesforce does not recommend installing upgrades directly in production organizations. Instead, install Grants Management in one of your free sandbox environments or a developer organization first.
+
+Once you've installed Grants Management, check out these resources:
```
Comment (melissabarber) [inline README.md]:
```suggestion
```

File: README.md [hunk 7]
```diff
@@ -0,0 +1,143 @@
...
+
+-   [Grants Management Documentation](https://powerofus.force.com/s/article/GM-Documentation)
```
Comment (melissabarber) [inline README.md]:
```suggestion
```

File: README.md [hunk 8]
```diff
@@ -0,0 +1,143 @@
...
+-   [Salesforce Partner Community - Grants Management Experts Group](https://partners.salesforce.com/_ui/core/chatter/groups/GroupProfilePage?g=0F94V000000LpYeSAK)
+
+Please visit the [Grants Management customer group](https://trailhead.salesforce.com/trailblazer-community/groups/0F94S000000GvrgSAC) on the Trailblazer Community for any questions about Grants Management.
```
Comment (melissabarber) [inline README.md]:
```suggestion
```

File: README.md [hunk 9]
```diff
@@ -0,0 +1,143 @@
...
+
+## Prerequisites for all installers
```
Comment (melissabarber) [inline README.md]:
```suggestion
```

File: README.md [hunk 10]
```diff
@@ -0,0 +1,143 @@
...
+
+-   Grants Management Permission Set Licenses (PSLs) and Customer Community Plus Login licenses must be provisioned in your Salesforce org. For help, work with your sales representative.
```
Comment (melissabarber) [inline README.md]:
```suggestion
```

File: README.md [hunk 11]
```diff
@@ -0,0 +1,143 @@
...
+-   [Install CumulusCI](https://cumulusci.readthedocs.io/en/latest/get_started.html) and [connect your org to CumulusCI](https://cumulusci.readthedocs.io/en/latest/connected_orgs.html).
```
Comment (melissabarber) [inline README.md]:
```suggestion
```

File: README.md [hunk 12]
```diff
@@ -0,0 +1,143 @@
...
+    -   See the [CumulusCI Setup](https://trailhead.salesforce.com/en/content/learn/modules/cumulusci-setup) Trailhead module for step by step instructions how to [Review Base Requirements and Install Visual Studio Code](https://trailhead.salesforce.com/en/content/learn/modules/cumulusci-setup/review-base-requirements-install-visual-studio-code), [Install CumulusCI](https://trailhead.salesforce.com/en/content/learn/modules/cumulusci-setup/install-cumulusci), and [Connect to CumulusCI](https://trailhead.salesforce.com/en/content/learn/modules/cumulusci-setup/set-up-github-and-connect-to-cumulusci).
```
Comment (melissabarber) [inline README.md]:
```suggestion
```

File: README.md [hunk 13]
```diff
@@ -0,0 +1,143 @@
...
+-   [Set Up My Domain](https://help.salesforce.com/s/articleView?id=sf.domain_name_setup.htm&type=5) in your Salesforce org.
```
Comment (melissabarber) [inline README.md]:
```suggestion
```

File: README.md [hunk 14]
```diff
@@ -0,0 +1,143 @@
...
+-   [Enable Digital Experiences](https://help.salesforce.com/s/articleView?id=sf.workotcom_quemgmtenable_communities.htm&type=5) in your Salesforce org.
```
Comment (melissabarber) [inline README.md]:
```suggestion
```

File: README.md [hunk 15]
```diff
@@ -0,0 +1,143 @@
...
+-   [Enable chatter](https://help.salesforce.com/s/articleView?id=sf.collab_enable.htm&type=5) for your Salesforce org.
```
Comment (melissabarber) [inline README.md]:
```suggestion
```

File: README.md [hunk 16]
```diff
@@ -0,0 +1,143 @@
...
+
+> **NOTE**: Make sure to complete all [Prerequisites for all installers](#Prerequisites-for-all-installers) before attempting any installation.
```
Comment (melissabarber) [inline README.md]:
```suggestion
```

File: README.md [hunk 17]
```diff
@@ -0,0 +1,143 @@
...
+
+# Installation errors
```
Comment (melissabarber) [inline README.md]:
```suggestion
```

File: README.md [hunk 18]
```diff
@@ -0,0 +1,143 @@
...
+
+To get help with any installation error, immediately execute the `cci error info` command in the terminal and go to [help.salesforce.com](https://help.salesforce.com/), find the Support & Services area and log a technical support ticket with Salesforce. Include “Grants Management” in the subject line and include the output of `cci error info` in your comment.
```
Comment (melissabarber) [inline README.md]:
```suggestion
```

File: README.md [hunk 19]
```diff
@@ -0,0 +1,143 @@
...
+| Install Outbound Funds NPSP Extension                                   | Package  | Optional |
+| Create the Funding Program Portal                                       | Other    | Required |
+| Publish the Funding Program Portal                                      | Other    | Required |
+| Install Funding Program Portal Navigation Menu                          | Metadata | Required |
+| Add Grantseeker Profile to Funding Program Portal                       | Other    | Required |
+| Grant Visibility to Community Users                                     | Metadata | Required |
+| Add Funding Program Portal Branding                                     | Metadata | Required |
+| Update Funding Program Portal Profile (Guest User) profile              | Metadata | Required |
+
+To install, open a terminal at this repository and execute the following command:
```
Comment (melissabarber) [inline README.md]:
```suggestion
```

File: README.md [hunk 20]
```diff
@@ -0,0 +1,143 @@
...
+
+```
+cci flow run customer_org_full --org <your connected Salesforce org>
```
Comment (melissabarber) [inline README.md]:
```suggestion
```

File: README.md [hunk 21]
```diff
@@ -0,0 +1,143 @@
...
+```
+
+## Install Grants Management
+
+This will install Grants Management into your org.
```
Comment (melissabarber) [inline README.md]:
```suggestion
This installs Grants Management into your org.
```

Comment (melissabarber) [reply README.md]: @spelak-salesforce is the difference between this installer and the previous one, Install Grants Management and Build Community, that this one doesn't do anything with the community? Do customers chose to install this or the previous on? Should we call that out here?

File: README.md [hunk 22]
```diff
@@ -0,0 +1,143 @@
...
+| Install Grants Management                                               | Package  | Required |
+| Deploy Grants Management: Enhanced Application (Sample) Flow            | Metadata | Required |
+| Deploy Create Application Flow                                          | Metadata | Required |
+| Deploy Grants Management: Multi-Section Application Form (Sample) Flows | Metadata | Required |
+| Deploy Profiles                                                         | Metadata | Required |
+| Deploy Submit Application Flow                                          | Metadata | Required |
+| Deploy Grants Management: Validate EIN (Sample) Flow                    | Metadata | Required |
+| Install Outbound Funds NPSP Extension                                   | Package  | Optional |
+
+To install, open a terminal at this repository and execute the following command:
```
Comment (melissabarber) [inline README.md]:
```suggestion
```

File: README.md [hunk 23]
```diff
@@ -0,0 +1,143 @@
...
+
+```
+cci flow run customer_org --org <your connected Salesforce org>
```
Comment (melissabarber) [inline README.md]:
```suggestion
```

File: README.md [hunk 24]
```diff
@@ -0,0 +1,143 @@
...
+```
+
+## Install Grants Management Flows
```
Comment (melissabarber) [inline README.md]:
```suggestion
## Grants Management Flows
```

File: README.md [hunk 25]
```diff
@@ -0,0 +1,143 @@
...
+-   [Install Grants Management and Build Community](#Install-Grants-Management-and-Build-Community)
+-   [Install Grants Management](#Install-Grants-Management)
+-   [Install Grants Management Flows](#Install-Grants-Management-Flows)
+    -   [Deploy Grants Management: Multi-Section Application Form (Sample) and subflow](#Deploy-Grants-Management-Multi-Section-Application-Form-Sample-and-subflow)
+    -   [Deploy Enhanced Application (Sample) Flow](#Deploy-Enhanced-Application-Sample-Flow)
+    -   [Deploy Create Application Flow](#Deploy-Create-Application-Flow)
+    -   [Deploy Submit Application Flow](#Deploy-Submit-Application-Flow)
+    -   [Deploy Grants Management: Validate EIN (Sample) Flow](#Deploy-Grants-Management-Validate-EIN-Sample-Flow)
+
+## Install Grants Management and Build Community
```
Comment (melissabarber) [inline README.md]:
```suggestion
## Grants Management and Build Community
```

File: README.md [hunk 26]
```diff
@@ -0,0 +1,143 @@
...
+| Add Funding Program Portal Branding                                     | Metadata | Required |
+| Update Funding Program Portal Profile (Guest User) profile              | Metadata | Required |
+
+To install, open a terminal at this repository and execute the following command:
+
+```
+cci flow run customer_org_full --org <your connected Salesforce org>
+```
+
+## Install Grants Management
```
Comment (melissabarber) [inline README.md]:
```suggestion
## Grants Management
```

File: README.md [hunk 27]
```diff
@@ -0,0 +1,143 @@
...
+
+To install, open a terminal at this repository and execute the following command:
+
+```
+cci flow run customer_org --org <your connected Salesforce org>
+```
+
+## Install Grants Management Flows
+
+Install or overwrite the selected unpackaged Grants Management flows in your org. After installation, you can access and activate earlier versions of previously-installed flows as needed. Be sure to create a backup of any changes before proceeding.
```
Comment (melissabarber) [inline README.md]:
```suggestion
```

File: README.md [hunk 28]
```diff
@@ -0,0 +1,143 @@
...
+
+> **Additional Prerequisites**
```
Comment (melissabarber) [inline README.md]:
```suggestion
```

File: README.md [hunk 29]
```diff
@@ -0,0 +1,143 @@
...
+>
+> -   Grants Management version 1.17 is the minimum version required to (re)install any unpackaged Grants Management flows.
```
Comment (melissabarber) [inline README.md]:
```suggestion
```

File: README.md [hunk 30]
```diff
@@ -0,0 +1,143 @@
...
+
+### Deploy Grants Management: Multi-Section Application Form (Sample) and subflow
```
Comment (melissabarber) [inline README.md]:
```suggestion
### Grants Management: Multi-Section Application Form (Sample) and subflow
```

File: README.md [hunk 31]
```diff
@@ -0,0 +1,143 @@
...
+
+To install or overwrite the Grants Management: Multi-Section Application Form (Sample) and subflow, open a terminal at this repository and execute the following command:
```
Comment (melissabarber) [inline README.md]:
```suggestion
The Grants Management: Multi-Section Application Form (Sample) flow shows a grantseeker an application with multiple sections and allows a grantseeker to save and return to the application. This flow is continuous until a grantseeker submits their application.
```

File: README.md [hunk 32]
```diff
@@ -0,0 +1,143 @@
...
+
+```
+cci task run deploy_multi_stage_app_flow --org <your connected Salesforce org>
```
Comment (melissabarber) [inline README.md]:
```suggestion
```

File: README.md [hunk 33]
```diff
@@ -0,0 +1,143 @@
...
+```
+
+### Deploy Enhanced Application (Sample) Flow
```
Comment (melissabarber) [inline README.md]:
```suggestion
### Enhanced Application (Sample) Flow
```

File: README.md [hunk 34]
```diff
@@ -0,0 +1,143 @@
...
+
+To install or overwrite the Deploy Enhanced Application (Sample) Flow, open a terminal at this repository and execute the following command:
```
Comment (melissabarber) [inline README.md]:
```suggestion
The Enhanced Application (Sample) flow condenses the default application experience using the Create Application flow from three screens to one screen. This is a sample flow and includes example application language, unmapped custom fields, and a file upload component within a single page.
```

File: README.md [hunk 35]
```diff
@@ -0,0 +1,143 @@
...
+
+```
+cci task run deploy_enhanced_application_flow --org <your connected Salesforce org>
```
Comment (melissabarber) [inline README.md]:
```suggestion
```

File: README.md [hunk 36]
```diff
@@ -0,0 +1,143 @@
...
+```
+
+### Deploy Create Application Flow
```
Comment (melissabarber) [inline README.md]:
```suggestion
### Create Application Flow
```

File: README.md [hunk 37]
```diff
@@ -0,0 +1,143 @@
...
+
+To install or overwrite the Create Application Flow, open a terminal at this repository and execute the following command:
```
Comment (melissabarber) [inline README.md]:
```suggestion
The Create Application flow automates how the application initially starts for your grantseeker. This involves retrieving information about the Funding Program and user’s Contact information, autofilling fields on the application, and creating the application record. Without this flow, a grantseeker can’t apply for a Funding Request without making permission updates to the Grantseeker Plus Login profile.
```

File: README.md [hunk 38]
```diff
@@ -0,0 +1,143 @@
...
+
+```
+cci task run deploy_funding_program_application_flow --org <your connected Salesforce org>
```
Comment (melissabarber) [inline README.md]:
```suggestion
```

File: README.md [hunk 39]
```diff
@@ -0,0 +1,143 @@
...
+```
+
+### Deploy Submit Application Flow
```
Comment (melissabarber) [inline README.md]:
```suggestion
### Submit Application Flow
```

File: README.md [hunk 40]
```diff
@@ -0,0 +1,143 @@
...
+
+To install or overwrite the Submit Application Flow, open a terminal at this repository and execute the following command:
```
Comment (melissabarber) [inline README.md]:
```suggestion
The Submit Application flow automates how a grantseeker’s application is submitted to an organization. The flow retrieves the application record, makes sure the user is both a site user and owner of the application, and updates the application before it is submitted. Without this flow, the grantseeker’s application can’t be submitted to your organization for review.
```

File: README.md [hunk 41]
```diff
@@ -0,0 +1,143 @@
...
+
+```
+cci task run deploy_submit_application_flow --org <your connected Salesforce org>
```
Comment (melissabarber) [inline README.md]:
```suggestion
```

File: README.md [hunk 42]
```diff
@@ -0,0 +1,143 @@
...
+```
+
+### Deploy Grants Management: Validate EIN (Sample) Flow
```
Comment (melissabarber) [inline README.md]:
```suggestion
### Grants Management: Validate EIN (Sample) Flow
```

File: README.md [hunk 43]
```diff
@@ -0,0 +1,143 @@
...
+
+To install or overwrite the Grants Management: Validate EIN (Sample) Flow, open a terminal at this repository and execute the following command:
```
Comment (melissabarber) [inline README.md]:
```suggestion
The Grants Management: Validate EIN (Sample) Flow allows your grants managers to search for and validate the EIN of an organization.```

File: README.md [hunk 44]
```diff
@@ -0,0 +1,143 @@
...
+
+```
+cci task run deploy_validate_ein_flow --org <your connected Salesforce org>
```
Comment (melissabarber) [inline README.md]:
```suggestion
```

File: README.md [hunk 45]
```diff
@@ -0,0 +1,143 @@
...
+
+> **NOTE**: Make sure to complete all [Prerequisites for all installers](#Prerequisites-for-all-installers) before attempting any installation.
+
+# Installation errors
+
+To get help with any installation error, immediately execute the `cci error info` command in the terminal and go to [help.salesforce.com](https://help.salesforce.com/), find the Support & Services area and log a technical support ticket with Salesforce. Include “Grants Management” in the subject line and include the output of `cci error info` in your comment.
+
+# Installers
+
+-   [Install Grants Management and Build Community](#Install-Grants-Management-and-Build-Community)
```
Comment (melissabarber) [inline README.md]:
```suggestion
-   [Grants Management and Build Community](#Install-Grants-Management-and-Build-Community)
```

File: README.md [hunk 46]
```diff
@@ -0,0 +1,143 @@
...
+-   [Install Grants Management](#Install-Grants-Management)
```
Comment (melissabarber) [inline README.md]:
```suggestion
-   [Grants Management](#Install-Grants-Management)
```

File: README.md [hunk 47]
```diff
@@ -0,0 +1,143 @@
...
+-   [Install Grants Management Flows](#Install-Grants-Management-Flows)
```
Comment (melissabarber) [inline README.md]:
```suggestion
-   [Grants Management Flows](#Install-Grants-Management-Flows)
```

File: README.md [hunk 48]
```diff
@@ -0,0 +1,143 @@
...
+    -   [Deploy Grants Management: Multi-Section Application Form (Sample) and subflow](#Deploy-Grants-Management-Multi-Section-Application-Form-Sample-and-subflow)
```
Comment (melissabarber) [inline README.md]:
```suggestion
    -   [Grants Management: Multi-Section Application Form (Sample) and subflow](#Deploy-Grants-Management-Multi-Section-Application-Form-Sample-and-subflow)
```

File: README.md [hunk 49]
```diff
@@ -0,0 +1,143 @@
...
+    -   [Deploy Enhanced Application (Sample) Flow](#Deploy-Enhanced-Application-Sample-Flow)
```
Comment (melissabarber) [inline README.md]:
```suggestion
    -   [Enhanced Application (Sample) Flow](#Deploy-Enhanced-Application-Sample-Flow)
```

File: README.md [hunk 50]
```diff
@@ -0,0 +1,143 @@
...
+    -   [Deploy Create Application Flow](#Deploy-Create-Application-Flow)
```
Comment (melissabarber) [inline README.md]:
```suggestion
    -   [Create Application Flow](#Deploy-Create-Application-Flow)
```

File: README.md [hunk 51]
```diff
@@ -0,0 +1,143 @@
...
+    -   [Deploy Submit Application Flow](#Deploy-Submit-Application-Flow)
```
Comment (melissabarber) [inline README.md]:
```suggestion
    -   [Submit Application Flow](#Deploy-Submit-Application-Flow)
```

File: README.md [hunk 52]
```diff
@@ -0,0 +1,143 @@
...
+    -   [Deploy Grants Management: Validate EIN (Sample) Flow](#Deploy-Grants-Management-Validate-EIN-Sample-Flow)
```
Comment (melissabarber) [inline README.md]:
```suggestion
    -   [Grants Management: Validate EIN (Sample) Flow](#Deploy-Grants-Management-Validate-EIN-Sample-Flow)
```

File: README.md (hunk 45 above)
Comment (melissabarber) [reply README.md]: @spelak-salesforce @sella24 Note: I also updated the section headers further down in the README. Will that affect the jump links in this list?

File: README.md (hunk 43 above)
Comment (melissabarber) [reply README.md]: @spelak-salesforce or @sella24 please check what I've written here for accuracy. The product documentation didn't have an explanation of this flow.

**Review by melissabarber — CHANGES_REQUESTED** [review]
@spelak-salesforce and @sella24 I heavily edited the README to more closely mirror how the information is presented in the SSH Express Setup README. I moved the installation instructions here: https://docs.google.com/document/d/1FgcfqxO_BRN26ORa5YuIzKQPDCM5PSKsgbp1VwUsPbk/edit and I need a tech edit of the Google doc to make sure it's still accurate after my minor edits.

File: README.md (hunk 45 above)
Comment (spelak-salesforce) [reply README.md]: @melissabarber  Probably.  Can you test the README after the changes are committed?

**Comment by spelak-salesforce** [comment]
@melissabarber , you asked if the README links to other sections were broken, and they were.  But it was easy to fix!  Links in Girtub Markdown are easy.   Just copy the header where you want to link to, and replace all non-word characters with a single dash `-`. 

### Examples

1. Header is `## Grants Management and Build Community`
    - so link is `[Grants Management and Build Community](#Grants-Management-and-Build-Community)`.
2. Header is `### Grants Management: Multi-Section Application Form (Sample) and subflow`
    - so link is `[any link label you like](#Grants-Management-Multi-Section-Application-Form-Sample-and-subflow)`.

File: cumulusci.yml [hunk 53]
```diff
@@ -9,10 +9,353 @@ project:
...
+        group: "Grants Management: Experience Site"
+        description: "Create a NetworkMemberGroup record for the community user profile"
+        class_path: cumulusci.tasks.salesforce.network_member_group.CreateNetworkMemberGroups
+        options:
+            network_name: Funding Program Portal
+            profile_names:
+                - Grantseeker Plus Login
+
+    # Verification checks
+    create_verificationitem_document_type:
```
Comment (sella24) [inline cumulusci.yml]: Do we need these tasks ? 
`create_verificationitem_document_type`,
`create_taxstatussearch_document_type`

**Review by sella24 — CHANGES_REQUESTED** [review]
All look good. 
One comment below on some unused tasks in cumulusci.yem

File: README.md [hunk 54]
```diff
@@ -0,0 +1,81 @@
...
+- [Grants Management Flows](#Grants-Management-Flows)
+  - [Grants Management: Multi-Section Application Form (Sample) and subflow](#Grants-Management-Multi-Section-Application-Form-Sample-and-subflow)
+  - [Enhanced Application (Sample) Flow](#Enhanced-Application-Sample-Flow)
+  - [Create Application Flow](#Create-Application-Flow)
+  - [Submit Application Flow](#Submit-Application-Flow)
+  - [Grants Management: Validate EIN (Sample) Flow](#Grants-Management-Validate-EIN-Sample-Flow)
+
+## Grants Management and Build Community
+
+This installs Grants Management into your org. The installer creates a Community with the Grantee Portal template and applies our recommended configurations.
```
Comment (melissabarber) [inline README.md]:
```suggestion
This installs the recommended setup of Grants Management into your org. In addition to installing Grants Management, this installer creates a Community with the Grantee Portal template and applies our recommended configurations.
```

File: README.md [hunk 55]
```diff
@@ -0,0 +1,81 @@
...
+- [Grants Management and Build Community](#Grants-Management-and-Build-Community)
+- [Grants Management](#Grants-Management)
+- [Grants Management Flows](#Grants-Management-Flows)
+  - [Grants Management: Multi-Section Application Form (Sample) and subflow](#Grants-Management-Multi-Section-Application-Form-Sample-and-subflow)
+  - [Enhanced Application (Sample) Flow](#Enhanced-Application-Sample-Flow)
+  - [Create Application Flow](#Create-Application-Flow)
+  - [Submit Application Flow](#Submit-Application-Flow)
+  - [Grants Management: Validate EIN (Sample) Flow](#Grants-Management-Validate-EIN-Sample-Flow)
+
+## Grants Management and Build Community
```
Comment (melissabarber) [inline README.md]:
```suggestion
## Grants Management and Funding Program Portal Community
```

File: README.md [hunk 56]
```diff
@@ -0,0 +1,81 @@
...
+| Install Outbound Funds NPSP Extension                                   | Package  | Optional |
+| Create the Funding Program Portal                                       | Other    | Required |
+| Publish the Funding Program Portal                                      | Other    | Required |
+| Install Funding Program Portal Navigation Menu                          | Metadata | Required |
+| Add Grantseeker Profile to Funding Program Portal                       | Other    | Required |
+| Grant Visibility to Community Users                                     | Metadata | Required |
+| Add Funding Program Portal Branding                                     | Metadata | Required |
+| Update Funding Program Portal Profile (Guest User) profile              | Metadata | Required |
+
+## Grants Management
```
Comment (melissabarber) [inline README.md]:
```suggestion
## Grants Management, Only
```

File: README.md [hunk 57]
```diff
@@ -0,0 +1,81 @@
...
+
+This installs Grants Management into your org.
```
Comment (melissabarber) [inline README.md]:
```suggestion
This installs Grants Management into your org and doesn't create the Funding Program Portal.
```

File: README.md [hunk 58]
```diff
@@ -0,0 +1,81 @@
+# Grants Management Express Setup
+
+This read-only repository contains CumulusCI automation and unmanaged metadata components used in the Grants Management MetaDeploy installer.
+
+Visit the [Grants Management customer group](https://trailhead.salesforce.com/trailblazer-community/groups/0F94S000000GvrgSAC) or [Salesforce Partner Community - Grants Management Experts Group](https://partners.salesforce.com/_ui/core/chatter/groups/GroupProfilePage?g=0F94V000000LpYeSAK) for any questions about this repository or Grants Management.
+
+# Installers
+
+- [Grants Management and Build Community](#Grants-Management-and-Build-Community)
+- [Grants Management](#Grants-Management)
```
Comment (melissabarber) [inline README.md]:
```suggestion
- [Grants Management, Only](#Grants-Management-Only)
```

File: README.md [hunk 59]
```diff
@@ -0,0 +1,81 @@
+# Grants Management Express Setup
+
+This read-only repository contains CumulusCI automation and unmanaged metadata components used in the Grants Management MetaDeploy installer.
+
+Visit the [Grants Management customer group](https://trailhead.salesforce.com/trailblazer-community/groups/0F94S000000GvrgSAC) or [Salesforce Partner Community - Grants Management Experts Group](https://partners.salesforce.com/_ui/core/chatter/groups/GroupProfilePage?g=0F94V000000LpYeSAK) for any questions about this repository or Grants Management.
+
+# Installers
+
+- [Grants Management and Build Community](#Grants-Management-and-Build-Community)
```
Comment (melissabarber) [inline README.md]:
```suggestion
- [Grants Management and Funding Program Portal Community](#Grants-Management-and-Funding-Program-Portal-Community)
```

**Review by melissabarber — APPROVED** [review]
@spelak-salesforce made a few other minor edits to the headers and descriptions of the two installers to help call out the differences you outlined in Slack. Otherwise looks good to go!

File: cumulusci.yml (hunk 53 above)
Comment (spelak-salesforce) [reply cumulusci.yml]: @sella24 , see https://salesforce-internal.slack.com/archives/C02R27B4V4H/p1639769086020700

**Comment by spelak-salesforce** [comment]
> its to the headers and descriptions of the two installers to help call out the differences you outlined in Slack. Otherwise looks good to go!

@melissabarber , requested changes are committed.  Thank you!

**Review by sella24 — APPROVED** [review]
Great PR Scott.

**Pull request merged**